test/test_benchmark.py

```diff
@@ -921,6 +955,36 @@ def output_parser(output):
       return float(re.search(r'Total elapsed: ([\d\.]+)', output).group(1))
     self.do_benchmark('matrix_multiply', read_file(test_file('matrix_multiply.cpp')), 'Total elapsed:', output_parser=output_parser, shared_args=['-I' + test_file('benchmark')])
 
+  @non_core
+  def test_sizes(self):
+    assert common.EMTEST_BENCHMARK_SIZE_FILE, 'Filename for measurements should be set.'
```

Comment:
Rather than an env var, how about always writing this to the same place under `out/test/` (or maybe `out/bench/`)?

Then the user would run `./test/runner.py benchmark.test_sizes` and then copy from that directory.

That seems a little more consistent with how we do other things, and avoids a new env var, which seems useful. But I don't feel strongly.

Reply:
I'm also currently using `EMTEST_BENCHMARK_SIZE_FILE` to only run the `EmscriptenBenchmarker`. I suppose I could modify the benchmarks to only include that one benchmarker within the test. Does that seem fine or did you have something else in mind?

Reply:
Oh, I was thinking `./test/runner.py benchmark.test_sizes` was enough to just run that one benchmark, but I guess we also need to only run it in the one benchmarker. I don't remember if benchmarks are given the benchmarker, but that should be available somehow, and then we could do something like
```python
  if not self.benchmarker.is_emscripten():
    self.skipTest('benchmarking only runs on emscripten')
```

Reply:
I changed it around a bit per your recommendation and Sam's. I write the file to the same spot always and now there's a new environment variable to explicitly say which benchmarkers you want to run.